src/main/java/org/fisco/bcos/web3j/abi/Constant.java

```diff
@@ -0,0 +1,12 @@
+package org.fisco.bcos.web3j.abi;
+
+import java.math.BigInteger;
+
+public class Constant {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Constant' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/fisco/web3sdk/pullRequest?prid=4732380)